src/NServiceBus.AcceptanceTests/BestPractices/When_subscribing_to_command_bestpractices_disabled_on_endpoint.cs

```diff
@@ -7,6 +7,7 @@
 
     public class When_subscribing_to_command_bestpractices_disabled_on_endpoint : NServiceBusAcceptanceTest
     {
+        [Ignore("not supported via code first API")]
```

Comment:
I think this is fine since endpoints that want to work around this can just add a custom convention that marks all `ICommands` as plan `IMessages`